recipes/qxlsx/all/conanfile.py

```diff
@@ -43,29 +38,31 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("qt/5.15.3")
+        self.requires("qt/5.15.5")
+
+    def layout(self):
+        cmake_layout(self)
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                destination=self._source_subfolder, strip_root=True)
+        get(self, **self.conan_data["sources"][self.version],
+            destination=self.source_folder, strip_root=True)
 
-    @functools.lru_cache(1)
-    def _configure_cmake(self):
-        cmake = CMake(self)
-        cmake.configure()
-        return cmake
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["QT_VERSION_MAJOR"] = Version(self.deps_cpp_info["qt"].version).major
+        tc.generate()
```

Comment:
```suggestion
        tc.generate()
        tc = CMakeDeps(self)
        tc.generate()
```
It will generate cmake_find and others